bigframes/ml/llm.py

```diff
@@ -48,7 +49,14 @@ def __init__(
         connection_name: Optional[str] = None,
     ):
         self.session = session or bpd.get_global_session()
-        self.connection_name = connection_name or self.session._bq_connection
+
+        connection_name = connection_name or self.session._bq_connection
```

Comment:
I thought about this a bit, we are resolving between 3 potential sources of connection:

1. provided by the user to llm, i.e. `connection_name` arg
2.  defined by the user in the session, i.e. `self.session._bq_connection`
3. the default defined in `clients.py`

How about we set the default in  `Session.__init__` like below?

```python
self._bq_connection = context.bq_connection or "bigframes-default-connection"
```

We are assuming other session defaults there, such as `self._location = "US"`.

Reply:
Oh yea, I did this way at first place, but then I found remote_function doesn't always have a session object. So I had to put the default away from session. Then we added default session... 

We can move it to session.